src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,229 @@
+        4. POSIX attributes for local override has been changed
+        5. The name and overriden name is found and POSIX attributes are updated
+    :customerscenario: False
+    """
+    provider.user("user1").add(
+        uid=999011, gid=999011, home="/home/user1", gecos="user", shell="/bin/bash", password="Secret123"
+    )
+
+    client.sssd.domain["ldap_id_mapping"] = "False"
+    client.sssd.config_apply()
+    client.sssd.start()
+    client.sss_override.user("user1").add(name="o-user1")
+    client.sssd.restart()
+
+    result_ssh1 = client.auth.ssh.password("user1", "Secret123")
+    result_ssh2 = client.auth.ssh.password("o-user1", "Secret123")
+    result_fqn_ssh1 = client.auth.ssh.password(f"user1@{client.sssd.default_domain}", "Secret123")
+    result_fqn_ssh2 = client.auth.ssh.password(f"o-user1@{client.sssd.default_domain}", "Secret123")
+
+    result_getent1 = client.tools.getent.passwd("o-user1")
+    client.sss_override.user("user1").add(name="o-user1", uid=999999, gid=888888, home="/home/o-user1")
+    result_getent2 = client.tools.getent.passwd("user1")
+    result_getent3 = client.tools.getent.passwd("o-user1")
+
+    client.sss_override.user("user1").delete()
+
+    assert result_ssh1
+    assert result_ssh2
+    assert result_fqn_ssh1
+    assert result_fqn_ssh2
+    assert result_getent1.uid == 999011
+    assert result_getent1.gid == 999011
+    assert result_getent2.uid == 999999
+    assert result_getent2.gid == 888888
+    assert result_getent2.home == "/home/o-user1"
+    assert result_getent3.uid == 999999
+    assert result_getent3.gid == 888888
+    assert result_getent3.home == "/home/o-user1"
+
+
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.topology(KnownTopology.AD)
+def test_sss_overrides__overriding_group_name_and_gid(client: Client, provider: GenericProvider):
+    """
+    :title: Locally overriding the name and GID of a group
+    :setup:
+        1. Create group "group1" with posix attributes defined
+        2. Start SSSD
+    :steps:
+        1. Create local override "group1" and name it "o-group1"
+        2. Query groups by the local override name
+        3. Override the GID for the group "group1"
+        4. Query groups by the override name
+    :expectedresults:
+        1. Group local override is created
+        2. Group is found by the overriden name "o-group1"
+        3. Local override POSIX attribute updated
+        4. Group is found by the overriden name "o-group1" and GID changed
+    :customerscenario: False
+    """
+    provider.group("group1").add(gid=999999)
+    client.sssd.domain["ldap_id_mapping"] = "False"
+    client.sssd.config_apply()
+    client.sssd.start()
+    client.sss_override.group("group1").add(name="o-group1")
+    client.sssd.restart()
+
+    result_getent1 = client.tools.getent.group("group1")
+    result_getent2 = client.tools.getent.group("o-group1")
+    client.sss_override.group("group1").add(name="o-group1", gid=888888)
```

Comment:
You can use `group = client.sss_override.group(...).add(...)` and then `group.delete()`